static/styles/themes/dark-theme.scss

```diff
@@ -465,18 +465,17 @@ textarea.form-control {
     color: black;
 }
 
-.opt-decoration {
-    &.analysis,
-    &.mixed {
-        background: #fdfd96 !important;
+.stack-usage {
```

Comment:
Random thought: Should we give this some ofther font?
Maybe so it looks like a codelens text? https://microsoft.github.io/monaco-editor/playground.html?source=v0.48.0#example-extending-language-services-codelens-provider-example

Reply:
Interesting! Does this look better?
![image](https://github.com/compiler-explorer/compiler-explorer/assets/73080/a3398142-1824-4a18-9ebd-ee7f24aeb0b6)


For opt-remarks I would be interested in making part of some comments text link to source, and I'm not sure if that is possible with InjectedText. Does codeLensProvider enable this?

Reply:
Ooh I prefer the font there? But also I'm cool with merging this and improving/tweaking later :)

Reply:
@mattgodbolt sorry, I don't understand: which font do you prefer?